File: client/src/types/api/PortfolioManager.ts
import type { Fund } from "./Fund";

export type PortfolioManager = {
    id: number;
    name: string;
    email: string;
    phone?: string | null;
    department?: string | null;
    funds_managed: number;
    funds?: Fund[]; // related_name="funds"
  };

File: server/portfolio/models/portfolio_manager.py
from django.db import models

class PortfolioManager(models.Model):
    name = models.CharField(max_length=255)
    email = models.EmailField(unique=True)
    phone = models.CharField(max_length=20, blank=True, null=True)
    department = models.CharField(max_length=100, blank=True, null=True)
    funds_managed = models.PositiveIntegerField(default=0)

    def __str__(self):
        return self.name
